src/content/docs/de/guides/deploy/buddy.mdx

```diff
@@ -25,4 +34,6 @@ Buddy selbst hostet deine Website nicht. Stattdessen hilft es dir, den Erstellun
    ```
 
 6. Füge eine Deployment-Aktion hinzu - es gibt viele zur Auswahl, du kannst sie in [Buddys Aktionskatalog](https://buddy.works/actions) durchsuchen. Obwohl die Einstellungen unterschiedlich sein können, musst du den **Quellpfad** auf `dist` setzen.
+
 7. Press the **Run** button.
```

Comment:
Looks like this step hasn't been translated! Could you add to the PR?

Reply:
omg, completely missed that, thx